src/test/wallet.ts

```diff
@@ -34,16 +11,17 @@ export const networkPortMap = {
 };
 
 export const getWalletData = (
-  network: Network
+  network: Network,
+  onFork: boolean
```

Comment:
```suggestion
  onFork = true
```
we can apply true as the default value, it would be compatible with the old code and wouldn't break the usage in other test files. 
currently it creates some typescript issues in other test files.